### PR TITLE
Chooser can be configured to allow read-only directories to be selected

### PR DESCRIPTION
**Pull request by OOPMan**
This commit is pretty simple. It adds an optional additional parameter that can be used to instruct the DirectoryChooser to allow selection of read-only directories.

### CODE REVIEW

**Comment by OOPMan** [comment]
Not sure why the CI build is failing, this works on my system.

File: library/src/main/java/net/rdrei/android/dirchooser/DirectoryChooserFragment.java [hunk 1]
```diff
@@ -48,9 +48,11 @@
     public static final String KEY_CURRENT_DIRECTORY = "CURRENT_DIRECTORY";
     private static final String ARG_NEW_DIRECTORY_NAME = "NEW_DIRECTORY_NAME";
     private static final String ARG_INITIAL_DIRECTORY = "INITIAL_DIRECTORY";
+    private static final String ARG_ALLOW_READ_ONLY_DIRECTORY = "ALLOW_READ_ONLY_DIRECTORY";
     private static final String TAG = DirectoryChooserFragment.class.getSimpleName();
     private String mNewDirectoryName;
     private String mInitialDirectory;
+    private Boolean mAllowReadOnlyDirectory;
```
Comment (passy) [inline library/src/main/java/net/rdrei/android/dirchooser/DirectoryChooserFragment.java]: Can we used an unboxed (lower-case) boolean here please?

**Comment by passy** [comment]
@OOPMan Sorry for the wait and thanks for the PR. This looks good, but I'd like to avoid the nullable type there if possible. I'll look into the CI issue now.

**Comment by OOPMan** [comment]
@passy Woops, my bad on the boxed type. My Java is kinda rusty (Mostly doing Python at the moment and using Scala for an Android projectr :-)

I'll push an update to that tomorrow morning. I also discovered I'm getting a weird issue running assembleRelease on my local box but it doesn't seem to be related to my code? Not sure if it's the same thing as the CI issue or something else. I can post the output if you like?

**Comment by passy** [comment]
@OOPMan Sure! The CI should be fixed now, so the next revision of this should hopefully be green. :)

**Comment by OOPMan** [comment]
Updated the pull request, btw.

**Comment by passy** [comment]
Excellent, thank you!

**Pull request merged**